### ISSUE TITLE
The repo does not seem to build

### ISSUE THREAD

**Issue by fiktor** [tonlabs/ton-labs-executor#2]
The error I'm getting when trying to build this is

```
error[E0432]: unresolved imports `ton_block::GasFlatPfx`, `ton_block::GasPrices`, `ton_block::GasPricesEx`
  --> .../ton-labs-executor-1c02eaf6bb19aa83/1f7e80b/src/blockchain_config.rs:17:5
```

See [20200710v2_build_sh.log](https://github.com/tonlabs/ton-labs-executor/files/4902045/20200710v2_build_sh.log) for more details.

It is not clear to me how is it supposed to work. Let's look e.g. at `ton_block::GasFlatPfx`.
* Indeed, `blockchain_config.rs` refers to `ton_block::GasFlatPfx`.
* [Cargo.lock](https://github.com/tonlabs/ton-labs-executor/blob/1f7e80bbf9ea85193b8e542affadd4220dd9fb56/Cargo.lock) suggests it should be defined in [github.com/tonlabs/ton-labs-block.git#d658c653cb65eb42ad4492e7604c939b560f7ff0](https://github.com/tonlabs/ton-labs-block.git#d658c653cb65eb42ad4492e7604c939b560f7ff0).
* Per [this git search](https://github.com/tonlabs/ton-labs-block/search?q=GasFlatPfx&unscoped_q=GasFlatPfx) it does not seem to be defined there.

**Comment by fiktor** [comment]
See also https://github.com/tonlabs/net.ton.dev/issues/30

**Comment by ASLanin** [comment]
Fixed with  #3
